crates/iota-sdk/examples/prepare_transfer_objects.rs

```diff
@@ -29,7 +29,7 @@ async fn main() -> Result<()> {
 
     let txn = builder.finish().await?;
 
-    println!("Signing Digest: {}", hex::encode(txn.signing_digest()));
+    println!("Signing Digest: {}", txn.signing_digest_hex());
     println!(
         "Txn Bytes: {}",
         base64ct::Base64::encode_string(&bcs::to_bytes(&txn)?)
```

Comment:
For this and all others, use `bcs_serialize_base64`?

Reply:
What do you mean done? I can still see it undone?

Reply:
ah, thought I did rust already but I just did the bindings